challenge.js

```diff
@@ -52,9 +57,10 @@
  *
  *	Use Google to search MDN for the appropriate method you can
  *		use to achieve this.
- *
+ *s
  *	Paste the URL of the MDN page below.
- */
+ */elector1 > selector2 { style properties }
+ https://developer.mozilla.org/en-US/docs/Web/CSS/Child_selectors
```

Comment:
You did get a useful MDN link for question 3. This doesn't satisfy question 4, though. 1/4